admin-dev/themes/new-theme/js/pages/product/edit/index.ts

```diff
@@ -72,7 +72,7 @@ $(() => {
     new CombinationsManager(productId);
   }
 
-  new NavbarHandler(ProductMap.navigationBar);
+  new NavbarHandler($(ProductMap.navigationBar));
```

Comment:
It worked without any issues because `$navigationContainer` is used as a string + inside a jQuery selector, both are working but we need to make a choice between using a jQuery element as a param or a string, I feel like it's way more readable to use a jQuery object on class instanciation because you know faster that you manipulate a jQuery element inside NavbarHandler